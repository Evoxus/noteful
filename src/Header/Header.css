.Header {
  background-color: #0b1f3a;
  color: #ebebeb;
  padding-bottom: 20px;
}

.Header a:visited {
  color: #ebebeb;
}

.Header a:hover {
  color: #8d8d8d
}