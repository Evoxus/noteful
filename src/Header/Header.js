import React from 'react';
import { Link } from 'react-router-dom';

export default function Header(props) {
  return (
    <header>
      <Link to='/'>Noteful</Link>
    </header>
  )
}