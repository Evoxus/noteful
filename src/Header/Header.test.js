import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { BrowserRouter } from 'react-router-dom';
import toJson from 'enzyme-to-json';
import Header from './Header';

describe('App component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<BrowserRouter> <Header /> </BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  })
  it('snapshot test', () => {
    const wrapper = shallow(<Header />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})