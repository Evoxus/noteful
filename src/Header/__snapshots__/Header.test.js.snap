// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component snapshot test 1`] = `
<header
  className="Header"
>
  <h1>
    <Link
      to="/"
    >
      Noteful
    </Link>
  </h1>
</header>
`;
